worker/src/lib.rs

```diff
@@ -105,12 +105,19 @@ pub async fn witness_instagram_post(
     ig_handle: String,
     ig_link: String,
     sig: String,
+    sig_type: String,
 ) -> Promise {
     future_to_promise(async move {
         let pk: JWK = jserr!(jwk_from_tezos_key(&public_key_tezos));
         let sk: JWK = jserr!(serde_json::from_str(&secret_key_jwk));
 
-        let mut vc = jserr!(instagram::build_instagram_vc_(&pk, &ig_handle));
+        // TODO: Add support for other signatures when expanding to Rebase:
+        let mut vc;
+        if sig_type == "tezos" {
```

Comment:
```suggestion
        let vc = if sig_type == "tezos" {
```

Not sure if it would be better with an enum + deserialisation, but I'm mentioning it in case you think it would be better.

Reply:
I'll change it to the `let vc = if` for now, I'd prefer an enum with deserialization, but I'm going to wait to implement that until we have other signatures set up. The important thing is that the route gets this information now so that the end point to FB's perspective doesn't change when things move over to Rebase.